app/views/cases/offender_sar_complaint/reopen.html.slim

```diff
@@ -0,0 +1,15 @@
+- content_for :heading, flush: true
+  = t('cases.offender_sar_complaint.reopen_reset_external_deadline_heading')
+
+= render partial: 'layouts/header'
+
+= form_for @case, url: confirm_reopen_case_sar_offender_complaint_path(@case), as: :offender_sar_complaint do |f|
+
+  .form-group
+    = f.gov_uk_date_field :external_deadline, { \
+        legend_text: '',
```

Comment:
Not sure how this renders, can you please double check this meets accessibility or let me know how to get there? Thanks.

Reply:
I didn't pay attention to this,  thank you for spotting this!   I think maybe I just add one? 
The steps for get this page is 
1 - create an complaint case
2 - proceed the case to close, 
3 - once the case is closed, you should see 'Reopen' button, click it you will see the page

Reply:
I added a screenshot for this PR, @digitalali-moj , that is how the deadline looks like

Reply:
I added the legend_text, the new looks like below,  let me know whether it is ok, thank you!
<img width="697" alt="Screenshot 2021-01-28 at 10 59 32" src="https://user-images.githubusercontent.com/26111671/106129206-f24b4400-6157-11eb-918a-f29f4c398d50.png">

Reply:
Looks great to me! thanks!